src/System.Management.Automation/engine/remoting/client/ClientMethodExecutor.cs

```diff
@@ -18,12 +18,12 @@ internal class ClientMethodExecutor
         /// <summary>
         /// Transport manager.
         /// </summary>
-        private BaseClientTransportManager _transportManager;
+        private readonly BaseClientTransportManager _transportManager;
 
         /// <summary>
         /// Client host.
         /// </summary>
-        private PSHost _clientHost;
+        private readonly PSHost _clientHost;
```

Comment:
Why not make Guid fields readonly as well?

Reply:
Maybe the IDE0044 analyzer doesn't suggest fixes for struct types, I will make changes a new PR

Reply:
No, please make called out changes here.

Reply:
I already started work in a new PR, now submitted as #14000.

Reply:
Ok, that will work.

Reply:
#14000 is ready for review.